projects/liqwid/index.js

```diff
@@ -67,16 +92,36 @@ const tokenMapping = {
 
 const getToken = market => tokenMapping[market.marketId.toUpperCase()] ?? base64ToHex(market.info.params.underlyingClass.value0.symbol)
 
+const getOptimBondTVL = async () => {
+  const getLoans = async (pageIndex) => {
+    const { Page: { pageInfo, loan: loans } } = await graphQuery(endpoint, queryAdaLoans, { page: pageIndex })
+
+    if (!pageInfo.hasNextPage) {
+      return loans
+    }
+    return [...loans, ...(await getLoans(pageIndex + 1))]
+  }
+  const loans = await getLoans(0)
+  const relevantLoans =
+    loans.filter(l => (l.collaterals.filter(c => c.id === "OptimBond1")).length != 0)
+  const bonds =
+    relevantLoans.map(l => l.collaterals[0].amount).reduce((acc, amount) =>
+      acc + Number(amount), 0)
+  const { Market: { price: { price: adaPrice } } } = await graphQuery(endpoint, queryAdaPrice)
+
+  return bonds * 100 * adaPrice
```

Comment:
If you remove now export just the quantity of bonds, rather than priced in USD, it should work

Reply:
Trying now